src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/pip/PipAwardType.java

```diff
@@ -1,14 +1,14 @@
-package uk.gov.hmcts.reform.sscs.ccd.presubmit.writefinaldecision;
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.writefinaldecision.pip;
 
 /**
  * Enum specifying the possible types of Award, along with the key for that award in CCD.
  */
-public enum AwardType {
+public enum PipAwardType {
```

Comment:
Is this not consistent across benefit types? How different are they?

Reply:
I've now merged these award types into a single enum